### PR TITLE
Miscellaneous improvements

### PR DESCRIPTION
**Pull request by wainersm**
This contains just miscellaneous improvements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ options use-vc
 
 * [libkrunfw](https://github.com/containers/libkrunfw)
 * A working [Rust](https://www.rust-lang.org/) toolchain
+* C Library static libraries, as the [init](init/init.c) binary is statically linked.
```
Comment (slp) [inline README.md:60]: Perhaps it'd be helpful to put a couple of examples here (`glibc-static` in Fedora, whatever is called in Debian...).

**Review by slp — COMMENTED** [review]
Thanks a lot for this PR. LGTM, I just commented on a minor suggestion.

**Review by slp — COMMENTED** [review]
Thanks a lot for this PR. LGTM, I just commented on a minor suggestion.

File: README.md (hunk 1 above)
Comment (wainersm) [reply README.md:60]: @slp agreed, it will be helpful. I will send a v2.

**Comment by wainersm** [comment]
@slp I just updated the commit f165ed79a25c171d591d09ab71d4f6ba2ad4db5e that address your comment. I put only the package for Fedora given that the static libc for ubuntu/debian is already in the regular package.

**Comment by slp** [comment]
Merged, thanks!

**Pull request merged**